doc/siterc.tex

```diff
@@ -853,3 +853,70 @@ \subsection{[authentication]}
     \end{myitemize}
 \item {\em default:} state-totals
 \end{myitemize}
+
+\subsubsection{[authentication] $\rightarrow$ hashes}
+
+This sets the hash algorithms used for Pyro HMAC. SHA-256 and SHA-512 are
```

Comment:
Could you add a sentence to explain what "Pyro HMAC" is, to the limited extent that cylc users need to understand it?